docs/pro/pricing.md

```diff
@@ -4,3 +4,271 @@ title: Alaveteli Pro Pricing
+all the professional features.
+
+## Payment gateway
+
+For a payment gateway Alaveteli Professional uses [Stripe](https://stripe.com).
+It is widely available around the world and is easy to configure and work with.
+
+The main advantage of Stripe is that all payment information is entered and sent
+directly to Stripe. They never gets transmitted to or via your Alaveteli
+installation so the security and data protection of those details are not a
+concern for you.
+
+### Creating an account
+
+Signing up for an account with Stripe is quick and easy. Initially your account
+will be a limited test account.
+
+Before you can take payments you will need to activate your account. You will
+need to provide legal information on you and/or your company as well as your
+bank details in order receive payments.
+
+Once approved by Stripe you'll then be able to receive payments and Stripe will
+transfer the money directly into your bank account.
+
+## Stripe configuration
+
+Stripe provides a secure and robust Application Programming Interface or API
+which Alaveteli Professional uses to communicate with Stripe. In order to use
+this we require some configuration setting from your Stripe dashboard.
+
+Follow the instructions below to set this up for you Alaveteli installation.
+
+* [API keys](#api-keys)
+* [Webhook endpoint](#webhook-endpoint)
+* [Setting up the subscription product](#setting-up-the-subscription-product)
+* [Creating coupons](#creating-coupons)
+* [Using a namespace](#using-a-namespace)
+
+### API keys
+
+For Alaveteli Professional to communicate securly with Stripe you'll need to
+generate a pair of [standard API keys](https://dashboard.stripe.com/apikeys) and
+add the [`STRIPE_PUBLISHABLE_KEY`](#stripe_publishable_key) and
+[`STRIPE_SECRET_KEY`](#stripe_secret_key) settings.
+
+### Webhook endpoint
+
+Stripe uses [webhooks](https://dashboard.stripe.com/webhooks) to inform the
+Alaveteli installation of important events, such as when a users bank card can't
+be charged or when a subscription has been cancelled.
+
+You should add a endpoint pointing to `/pro/subscriptions/stripe-webhook` for
+your Alaveteli installation.
+
+The event types which should be subscribed to are:
+* `customer.subscription.deleted`
+* `invoice.payment_succeeded`
+* `invoice.payment_failed`
+* `customer.subscription.updated`
+
+Finally you'll set the [`STRIPE_WEBHOOK_SECRET`](#stripe_webhook_secret) to the
+a signing secret found in the Stripe dashboard.
+
+### Setting up the subscription product
+
+A subscription product in Stripe need to be [created](
+  https://dashboard.stripe.com/subscriptions/products/create
+) this can have any name you wish.
+
+Once a product is created you'll then be asked to set-up its the pricing plan.
+The pricing plan should be have the ID of `pro`, be for a recurring quantity and
+without multiple price tiers.
+
+It's will be up to you to decide the price per unit, billing interval and trial
+period.
```

Comment:
Not 100% sure about trial period. I _think_ it would be okay because in this case it should ask for card up front? We don't explicitly handle trial ending though. This might be worth checking for suitability.

Reply:
From my tests we don't need to do anything to support Stripe trial periods. It's different to how we originally managed trials as its all managed by Stripe. They take care of everything and we receive all necessary webhooks already.

Have changed the billing interval wording tho.